src/components/dashboard/GroupList.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { Avatar, List } from 'antd';
 import PropTypes from 'prop-types';
 
-function GroupList({ groups, onGroupClick }) {
+function GroupList({ groups, index, onGroupClick }) {
```

Comment:
I think `activeIndex` is probably a more clear prop name here.

Reply:
Changed the name